package.json

```
@@ -75,8 +72,7 @@
   "scripts": {
     "all": "npm run lint && npm run test && npm run accept",
     "babel-watch": "babel src -d dist --watch --source-maps --copy-files",
-    "coverage": "istanbul cover ./node_modules/mocha/bin/_mocha -- -R spec test/ --compilers js:babel-register",
-    "coveralls": "istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R spec --compilers js:babel-register test/ && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage",
+    "coverage": "istanbul cover _mocha ./test/**/*.{integration,spec}.js -- --compilers js:babel-register",
```

For nyc, see https://github.com/particle-iot-inc/app-metrics/blob/main/package.json#L22
```suggestion
    "coverage": "nyc --reporter=html npm test",
```